code/modules/mob/living/basic/jungle/jungle_mobs.dm

```diff
@@ -0,0 +1,13 @@
+/mob/living/basic/jungle
+	habitable_atmos = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	faction = list(FACTION_JUNGLE)
+	obj_damage = 30
+	environment_smash = ENVIRONMENT_SMASH_WALLS
+	minimum_survivable_temperature = T0C
+	maximum_survivable_temperature = T0C + 450
+	status_flags = NONE
+	// Let's do a blue, since they'll be on green turfs if this shit is ever finished
+	lighting_cutoff_red = 5
+	lighting_cutoff_green = 20
+	lighting_cutoff_blue = 25
+	mob_size = MOB_SIZE_LARGE
```

Comment:
I'm not a fan of porting over this type of the framework from simple animals. It's an artifact that should really be kept there IMO. I think that we should just strike out the `jungle` subpath and just move all these variables to the mega arachnid itself. Having this typepath makes me feel weird because I have fears that people will inappropriately extend this typepath in the future.

Also, the jungle rework is likely never going to happen (all projects are vaporware until a PR is posted and merged). It doesn't make sense to keep the nomenclature around because it might inhibit someone from actually adding these really cool mobs to already-extant parts of the game (like away missions, ruins, etc.) since it's "not jungley".